docs/environment-docs.yml

```diff
@@ -25,3 +25,4 @@ dependencies:
       # See: https://github.com/geojupyter/jupytergis/issues/585
       - ../python/jupytergis_core
       - ../python/jupytergis_lab
+      - ../python/jupytergis
```

Comment:
I believe this is the culprit for the failure in the docs. We can't have `jupytergis` installed in the lite deployment of the docs.

It needs to be `jupytergis_lite` or nothing.